tests/db/dumps/v202209160.sql

```diff
@@ -0,0 +1,1111 @@
+--
+-- PostgreSQL database dump
+--
+
+-- Dumped from database version 12.12 (Ubuntu 12.12-0ubuntu0.20.04.1)
+-- Dumped by pg_dump version 12.12 (Ubuntu 12.12-0ubuntu0.20.04.1)
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SET check_function_bodies = false;
+SET xmloption = content;
+SET client_min_messages = warning;
+SET row_security = off;
+
+--
+-- Name: change; Type: TYPE; Schema: public; Owner: -
+--
+
+CREATE TYPE public.change AS ENUM (
+    'nochange',
+    'created',
+    'purged',
+    'updated'
+);
+
+
+--
+-- Name: non_deploying_resource_state; Type: TYPE; Schema: public; Owner: -
+--
+
+CREATE TYPE public.non_deploying_resource_state AS ENUM (
+    'unavailable',
+    'skipped',
+    'dry',
+    'deployed',
+    'failed',
+    'available',
+    'cancelled',
+    'undefined',
+    'skipped_for_undefined',
+    'processing_events'
+);
+
+
+--
+-- Name: notificationseverity; Type: TYPE; Schema: public; Owner: -
+--
+
+CREATE TYPE public.notificationseverity AS ENUM (
+    'message',
+    'info',
+    'success',
+    'warning',
+    'error'
+);
+
+
+--
+-- Name: resourceaction_type; Type: TYPE; Schema: public; Owner: -
+--
+
+CREATE TYPE public.resourceaction_type AS ENUM (
+    'store',
+    'push',
+    'pull',
+    'deploy',
+    'dryrun',
+    'getfact',
+    'other'
+);
+
+
+--
+-- Name: resourcestate; Type: TYPE; Schema: public; Owner: -
+--
+
+CREATE TYPE public.resourcestate AS ENUM (
+    'unavailable',
+    'skipped',
+    'dry',
+    'deployed',
+    'failed',
+    'deploying',
+    'available',
+    'cancelled',
+    'undefined',
+    'skipped_for_undefined',
+    'processing_events'
+);
+
+
+--
+-- Name: versionstate; Type: TYPE; Schema: public; Owner: -
+--
+
+CREATE TYPE public.versionstate AS ENUM (
+    'success',
+    'failed',
+    'deploying',
+    'pending'
+);
+
+
+SET default_tablespace = '';
+
+SET default_table_access_method = heap;
```

Comment:
```suggestion
--SET default_table_access_method = heap;
```
See https://github.com/inmanta/inmanta-core/blob/19bac9ed9c0790522a7ef0c985c464d90c74db1a/tests/db/dump_tool.py#L44